loge.js

```diff
@@ -59,16 +61,27 @@
     }
 
     if (isChrome) {
-      completeResult = results.join('');
+      var temp = results;
       results = [];
-      results.push('%c%s');
-      results.push('font-family: Comic Sans MS; line-height: 2em; font-size: 1.5em; color: ' + randomColor());
-      results.push(completeResult);
+      var format = '';
+      for(var i = 0; i < temp.length; i++){
+        format = format + '%c%s';
+      }
+
+      results.push(format);
+      
+      for(var i = 0; i < temp.length; i++){
```

Comment:
following the surrounding code, the format of this should be `for (var i = 0; i < temp.length; i++) {`